bin/anthology/texmath.py

```diff
@@ -19,7 +19,7 @@
 import logging as log
 import os
 import TexSoup
-
+from TexSoup.data import *
```

Comment:
Let's not use star imports, better to list all the names we need.

Reply:
done